aes/src/ni/utils.rs

```diff
@@ -1,15 +1,17 @@
-#[cfg(test)]
-use super::arch::__m128i;
-#[cfg(test)]
-use core::mem;
+//! Utility functions
 
+use super::arch::*;
 use cipher::{
     consts::{U16, U8},
     generic_array::GenericArray,
 };
 
+#[cfg(test)]
+use core::mem;
```

Comment:
You can remove this import and instead use `core::mem::transmute` in the `check` function.